Upstream-Status: Inappropriate [manually created]
--- git.orig/config.conf.example
+++ git/config.conf.example
@@ -1,6 +1,6 @@
 [client]
 # host or IP and optional port
-hawkbit_server            = 10.10.0.254:8080
+hawkbit_server            = 192.168.1.34:8080
 
 # true = HTTPS, false = HTTP
 ssl                       = false
@@ -12,10 +12,10 @@
 tenant_id                 = DEFAULT
 
 # Target name (controller id)
-target_name               = test-target
+target_name               = test-target-raspi4
 
 # Security token
-auth_token                = cb115a721af28f781b493fa467819ef5
+auth_token                = 04e3c792bf57abc7028cae718effb43d
 
 # Or gateway_token can be used instead of auth_token
 #gateway_token             = cb115a721af28f781b493fa467819ef5
